docs/src/model_providers_and_supported_functions.md

```diff
@@ -0,0 +1,30 @@
+```@meta
+CurrentModule = PromptingTools
+```
+
+# Model Providers and Supported Functions
+
+PromptingTools.jl routes AI calls through the use of subtypes of AbstractPromptSchema, which determine how data is formatted and where it is sent. (For example, OpenAI models have the corresponding subtype AbstractOpenAISchema, having the corresponding schemas - OpenAISchema, CustomOpenAISchema, etc.) This ensures that the data is correctly formatted for the specific AI model provider. 
+
+Below is an overview of the model providers supported by PromptingTools.jl, along with the corresponding schema information.
+
+| Abstract Schema         | Schema                    | Model Provider                         | aigenerate | aiembed | aiclassify | aiextract | aiscan | aiimage |
+|-------------------------|---------------------------|----------------------------------------|------------|---------|------------|-----------|--------|---------|
+| AbstractOpenAISchema    | OpenAISchema              | OpenAI                                 | ✅         | ✅     | ✅         | ✅       | ✅     | ✅     |
+| AbstractOpenAISchema    | CustomOpenAISchema*       | Any OpenAI-compatible API (eg, vLLM)*  | ✅         | ✅     | ❌         | ✅       | ✅     | ❌     |
+| AbstractOpenAISchema    | LocalServerOpenAISchema** | Any OpenAI-compatible Local server**   | ✅         | ✅     | ❌         | ✅       | ✅     | ❌     |
+| AbstractOpenAISchema    | MistralOpenAISchema       | Mistral AI                             | ✅         | ✅     | ❌         | ✅       | ✅     | ❌     |
+| AbstractOpenAISchema    | DatabricksOpenAISchema    | Databricks                             | ✅         | ✅     | ❌         | ✅       | ✅     | ❌     |
+| AbstractOpenAISchema    | FireworksOpenAISchema     | Fireworks AI                           | ✅         | ✅     | ❌         | ✅       | ✅     | ❌     |
+| AbstractOpenAISchema    | TogetherOpenAISchema      | Together AI                            | ✅         | ✅     | ❌         | ✅       | ✅     | ❌     |
+| AbstractOllamaSchema    | OllamaSchema              | Ollama                                 | ✅         | ✅     | ❌         | ❌       | ✅     | ❌     |
```

Comment:
See the discussion about difference in Ollama schemas explained here: https://github.com/svilupp/PromptingTools.jl/issues/68

I think it would be great to add which endpoint they hit, eg, `Ollama (endpoint `api/chat`)` or something like that. 

What do you think?